package.json

```diff
@@ -135,6 +135,7 @@
     "through2": "^2.0.3",
     "uglifyjs-webpack-plugin": "^1.1.6",
     "webpack": "^4.30.0",
+    "webpack-bundle-analyzer": "^3.3.2",
```

Comment:
Please remove

Reply:
Why? i will hook it to an npm script instead

Reply:
We can also write a special job based on that for CI to check if index bundle grow over our desired size and alert on PRs

Reply:
That already exits, size-buddy is called or something like that.
I mean remove it because it is not used by anything by the time I wrote my comment

Reply:
This one https://github.com/siddharthkp/bundlesize

Reply:
in the last commit now